zio-http/src/test/scala/zhttp/http/EncodeClientParamsSpec.scala

```diff
@@ -9,11 +9,12 @@ import zio.test._
 
 object EncodeClientParamsSpec extends DefaultRunnableSpec with EncodeClientParams {
 
-  val anyClientParam: Gen[Random with Sized, Client.ClientParams] = HttpGen.clientParams(
+  val anyClientParam: Gen[Random with Sized, Client.ClientParams]         = HttpGen.clientParams(
     HttpGen.httpData(
       Gen.listOf(Gen.alphaNumericString),
     ),
   )
+  val clientParamForFileData: Gen[Random with Sized, Client.ClientParams] = HttpGen.clientParamsForFileHttpData()
```

Comment:
Can inline this value.

Reply:
On second thoughts, may be Gen is not required. We are always reading a file from `resources` dir.